nearswap/src/deposit.rs

```diff
@@ -59,6 +59,24 @@ impl NearSwap {
         env_log!("Deposit, {} yNEAR", amount);
     }
 
+    /// Record deposit of some number of tokens to this contract.
+    /// Fails if account is not registered or if token isn't whitelisted.
+    pub(crate) fn deposit_token(
+        &mut self,
+        sender_id: &AccountId,
+        token_id: &AccountId,
+        amount: Balance,
+    ) {
+        let mut account_deposit = self.get_deposit(sender_id);
```

Comment:
```suggestion
        let mut d = self.get_deposit(sender_id);
```
let's use short names

Reply:
ok